listenbrainz/webserver/static/js/src/listens/ListenCountCard.tsx

```diff
@@ -9,15 +9,17 @@ const ListenCountCard = (props: ListenCountCardProps) => {
   const { listenCount } = props;
 
   return (
-    <Card id="listen-count-card">
-      <h4>
-        You have listened to
-        <hr />
-      </h4>
-      <div>
-        <h4>{listenCount}</h4>
-        <small className="text-muted">songs so far</small>
-      </div>
+    <Card id="listen-count-card" style={{ fontSize: 24 }}>
```

Comment:
It would be better to define font size using css instead of inline.
The `listen-count-card` id makes that easy.